.github/workflows/unit-tests.yml

```diff
@@ -16,6 +16,7 @@ jobs:
   build-and-unit-tests:
     name: "Build and unit tests"
     runs-on: self-hosted
+    if: "!contains(github.event.head_commit.message, '[skip ci]') || !contains(github.event.head_commit.message, '[ci skip]') || !contains(github.event.head_commit.message, '[no ci]') || !contains(github.event.head_commit.message, '[skip actions]') || !contains(github.event.head_commit.message, '[actions skip]')"
```

Comment:
We need to add this `if:` to all other workflows.
I would keep `[skip ci]` term only to reduce the variety of the options.